ReactNativeClient/root.js

```diff
@@ -660,6 +624,51 @@ class AppComponent extends React.Component {
 		});
 
 		AppState.addEventListener('change', this.onAppStateChange_);
+
+		const sharedData = Platform.OS === 'android' ? await ShareExtension.data() : null;
```

Comment:
Here also the platform check is not needed.

Reply:
Rremoved